index.html

```diff
@@ -513,6 +519,28 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
 =&gt; [4, 5, 2, 3, 0, 1]
 </pre>
 
+<p id="transform">
+        <b class="header">transform</b><code>_.transform(collection, iteratee, [memo], [context])</code>
+        <br />
+        <tt>_.transform</tt> is an alternative to <tt>_.reduce</tt> that transforms
+        the given collection into a new collection. Unlike <tt>_.reduce</tt> if the
+        memo isn't specified it will default to a new instance of the type of the collection.
+        Another distinction is that the memo cannot be changed, which means returning
```

Comment:
"can't be changed" -> "is immutable"

"means returning" -> "meaning it's unnecessary to return the memo from the iteratee."